x/participation/types/params.go

```diff
@@ -107,8 +132,41 @@ func validateParticipationTierList(v interface{}) error {
 		return fmt.Errorf("invalid parameter type: %T", v)
 	}
 
-	// TODO implement validation
-	_ = participationTierList
+	tiersIndexMap := make(map[uint64]struct{})
+	lastTierID := uint64(0)
+	lastRequiredAllocations := uint64(0)
+	lastTierBenefits := TierBenefits{MaxBidAmount: sdk.ZeroInt()}
+	for _, tier := range participationTierList {
```

Comment:
So if a TierList is a []Tier and must be sorted, do we really need the TierID?  Couldn't we just use the index in the slice as the ID?

Reply:
In my opinion, it gives more flexibility and make it a bit more reliable to explicitly define IDs
We can add a new tier in between without modifying the ID of the other tiers. In the front-end the tiers would be sorted by the required staking amount

Reply:
fair question. this is up for debate, as well as the validation logic I implemented. happy to discuss possible refinements.

Reply:
agreed with @lubtd